### ISSUE TITLE
CI builds are no longer running

### ISSUE THREAD

**Issue by ejd** [hozn/coilmq#31]
Would you be interested in a PR that fixes CI for coilmq?

CI builds are not running anymore because this project (as far as I can tell) isn't yet configured to have CI run on travis-ci.com.  travis-ci.org [no longer does builds](https://blog.travis-ci.com/2021-05-07-orgshutdown).

Unfortunately, the builds would fail even if they were running on the new travis-ci site.  I forked coilmq to work on #29 and enabled travis-ci for the fork.  [My results](https://app.travis-ci.com/github/ejd/coilmq) look pretty much the same [as when CI ran most recently against your repo](https://travis-ci.org/github/hozn/coilmq) for #28.

Pinning the fakeredis dependency to an earlier version may fix the problem with the python 2.7 build failures.  It may require listing a package multiple times in one of the requirements files and adding different [environment markers](https://www.python.org/dev/peps/pep-0508/#environment-markers) to each.  For example, `fakeredis < X.Y.Z; python_version <= '2.7'` and `fakeredis >= X.Y.Z; python_version > '2.7'`.  If a quick fix like these can't be made to work, I could set you up with tox -- a little more involved but, IMHO, very worthwhile.

**Comment by hozn** [comment]
@ejd , sure, I'd love it!  Also, I'd probably get rid of python 2.x support if I were looking at this code again.

Do you want me to just make you a maintainer?

**Comment by ejd** [comment]
I agree that it makes sense to drop Python 2.x support.  I'm looking forward to helping out!

**Comment by hozn** [comment]
Thanks so much, Eric.  I love that someone has taken interest in this
science project :). Feel free to take this in whatever direction you think
makes sense.
